store/user.js

```diff
@@ -9,11 +15,35 @@ class User {
 
   email = null
 
-  retrieveUser = async () => {}
+  url = null
+
+  organisationId = null
+
+  affiliationUrl = null
+
+  permissionsUrl = null
 
   constructor(rootStore) {
     this.rootStore = rootStore
   }
+
+  @action
+  logIn = async () => {
+    await this.retrieveUser()
+  }
+
+  @action
+  retrieveUser = async () => {
+    const user = await apiRequest('/user', 'GET', {}, {}, true)
+    this.rootStore.changeDataProvider(user.organisationId)
+    Object.assign(this, user)
```

Comment:
I am considering this being bad pattern but I am too lazy to do in other way. So, 👍